CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Add `BulletList` component.
+- Feature: Add `big` modifier to `BulletList`.
```

Comment:
Peut-être plutôt "to `.k-BulletList`." pour le distinguer de la ligne précédente ?